src/anurag/Array.java

```diff
@@ -0,0 +1,98 @@
+package anurag;
+
+public class Array {
+
+	int ar[] = { 10, 5, 30, 20, 90 };
```

Comment:
wrong thing, we need to pass array in method,
Always create parametrized method